mainnet-canisters.json

```diff
@@ -8,16 +8,16 @@
     "sha256": "612410c71e893bb64772ab8131d77264740398f3932d873cb4f640fc257f9e61"
   },
   "ck_btc_ledger": {
-    "rev": "d4ee25b0865e89d3eaac13a60f0016d5e3296b31",
-    "sha256": "a170bfdce5d66e751a3cc03747cb0f06b450af500e75e15976ec08a3f5691f4c"
+    "rev": "b98f0feed23702b954e61912e81c1613c075cffd",
+    "sha256": "0f194ee27c3a046423b5d36725df71b62156cec173a6c52890fcc2ac2308bbbf"
   },
   "ck_eth_index": {
     "rev": "d4ee25b0865e89d3eaac13a60f0016d5e3296b31",
     "sha256": "de250f08dc7e699144b73514f55fbbb3a3f8cd97abf0f7ae31d9fb7494f55234"
   },
   "ck_eth_ledger": {
-    "rev": "d4ee25b0865e89d3eaac13a60f0016d5e3296b31",
-    "sha256": "e6072806ae22868ee09c07923d093b1b0b687dba540d22cfc1e1a5392bfcca46"
+    "rev": "b98f0feed23702b954e61912e81c1613c075cffd",
```

Comment:
Is this actually what these canisters are running on mainnet?

Reply:
Hm - it actually looks like no, not yet.  The tests that we have to make sure upgrades are safe from mainnet versions rely on this file being correct.

Reply:
Once this file is updated to refer to canisters that contain https://github.com/dfinity/ic/pull/1969 we can revert the change in this file. I only need it for testing - V3 expects V2 (the above PR) to be the mainnet.

Reply:
I created [PR 2547](https://github.com/dfinity/ic/pull/2547) to update `mainnet-canisters.json` with the versions now actually running on mainnet.

Reply:
PR 2574 was merged and this PR doesn't modify `mainnet-canisters.json` anymore.